kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -96,6 +94,16 @@ TableInfo constructTableInfo(TableId table, Schema kafkaKeySchema, Schema kafkaV
     return tableInfoBuilder.build();
   }
 
+  public Map<TopicAndRecordName, Schema> discoverSchemas() {
```

Comment:
About the `Schema` - you're absolutely right, this was introduced before `String` was replaced with `TopicAndRecordName`. Since the latter provides all the necessary informations, `Schema` is no longer needed.

Reply:
BTW - the whole method will be removed as we are going to create tables on the fly.